cloudinit/distros/__init__.py

```diff
@@ -104,18 +111,22 @@ class Distro(persistence.CloudInitPickleMixin, metaclass=abc.ABCMeta):
     # This is used by self.shutdown_command(), and can be overridden in
     # subclasses
     shutdown_options_map = {"halt": "-H", "poweroff": "-P", "reboot": "-r"}
+    net_ops = iproute2
```

Comment:
This net_ops work feels like a bolt-on ideal that represents a bit of scope creep for this PR. Would it be okay to separate this to another PR to ease review? Otherwise, we can spend a bit more time going through this in this PR but I worry a bit about missing something here in the refactor dance.

Reply:
That's fair. I'll turn this into a 3 part series.